File: app/src/main/java/com/intelegain/agora/model/CommonStatusMessage.java
package com.intelegain.agora.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by maninder on 17/11/17.
 */

public class CommonStatusMessage {
    @SerializedName("Status")
    @Expose
    public String status;
    @SerializedName("Message")
    @Expose
    public String message;
}

File: app/src/main/java/com/intelegain/agora/model/CommonStatusMessage.kt
package com.intelegain.agora.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by maninder on 17/11/17.
 */
class CommonStatusMessage {
    @kotlin.jvm.JvmField
    @SerializedName("Status")
    @Expose
    var status: String? = null
    @kotlin.jvm.JvmField
    @SerializedName("Message")
    @Expose
    var message: String? = null
}